backend/controllers/messageController.js

```diff
@@ -38,8 +71,10 @@ const getMessage = async (req, res) => {
 const createMessage = async (req, res) => {
     const {name, email, mssg, author} = req.body
 
+    const idx = uuidv4();

```

Comment:
I don't like id ('9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d') too long
Can you use increase id (000001, 000002, ...., 999999) ?